src/openvic-simulation/history/CountryHistory.cpp

```diff
@@ -1,12 +1,13 @@
 #include "CountryHistory.hpp"
 
 #include "openvic-simulation/GameManager.hpp"
+#include <string_view>
 
 using namespace OpenVic;
 using namespace OpenVic::NodeTools;
 
 CountryHistoryEntry::CountryHistoryEntry(Country const& new_country, Date new_date)
-	: HistoryEntry { new_date }, country { new_country } {}
+	: HistoryEntry { new_date }, country { new_country }, releasable_vassal { true } {}
```

Comment:
```suggestion
	: HistoryEntry { new_date }, country { new_country } {}
```
`releasable_vassal` is a `std::optional<bool>`, so we only want it to be set to anything if it shows up in the history entry. There'll be an equivalent variable in either `Country` or `CountryInstance` which defaults to true, but here it should default to `std::nullopt_t` (which it will do if you don't explicitly initialise it).

Reply:
oh, right, didn't think of that